kubernetes/etc/kubernetes/k8s.node.rc

```diff
@@ -4,6 +4,6 @@ stuff "until nc -z __MASTER_IP__ 8080; do sleep 10; done; /home/ubuntu/kubernete
 screen -t kube-proxy
 select 1
 stuff "until nc -z __MASTER_IP__ 8080; do sleep 10; done; /home/ubuntu/kubernetes/cluster/ubuntu/binaries/minion/kube-proxy --master=http://__MASTER_IP__:8080^M"
-screen -t network-policy
+screen -t network-policy-agent
 select 2
-stuff "until curl -s http://__MASTER_IP__:8080/apis/romana.io/demo/v1/namespaces/default/networkpolicys/ | grep -q policy; do sleep 10; done; python /home/ubuntu/romana/kubernetes/k8s-listener.py^M"
+stuff "until curl -s http://__MASTER_IP__:8080/apis/romana.io/demo/v1/namespaces/default/networkpolicys/ | grep -q policy; do sleep 10; done; python /home/ubuntu/romana/kubernetes/k8s-listener.py --agent^M"
```

Comment:
If there is no one listening on target port i would expect listener to fail immediately, consider if you typed url incorrectly you don't want listener to sit quietly and wait for someone to show up.

Handling occasional dropouts is a different story - it must be implemented in listener but it won't cover for this use-case.